src/shared/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,36 @@
+import { NavLink } from 'react-router-dom';
+
+import IconAdd from '@components/svg/IconAdd';
+import IconHome from '@components/svg/IconHome';
+import IconUsers from '@components/svg/IconUsers';
+
+import './Navbar.scss';
+
+function Navbar() {
+  return (
+    <>
+      <div className="shadow-nav" />
+      <div className="navigation">
+        <ul>
+          <li>
+            <NavLink to="/" className={({ isActive }) => (isActive ? 'active' : 'not-active')}>
+              <IconHome />
+            </NavLink>
+          </li>
+          <li>
+            <NavLink to="/users" className={({ isActive }) => (isActive ? 'active' : 'not-active')}>
+              <IconUsers />
+            </NavLink>
```

Comment:
Tu peux utiliser la constante de PATHS et factoriser en créant une fonction `handleActiveClass` par exemple.

Pour ta fonction ne retourne rien quand l'élément n'est pas actif :
```js
function handleActiveClass({isActive}) {
 if (!isActive) return;

 return 'active';
}

```

```suggestion
            <NavLink to={PATHS.HOME} className={handleActiveClass}>
              <IconHome />
            </NavLink>
          </li>
          <li>
            <NavLink to={PATHS.PEOPLES} className={handleActiveClass}>
              <IconUsers />
            </NavLink>
```